ci/pipeline.yaml

```diff
@@ -233,3 +248,64 @@ jobs:
                 ! diff --context=3 \
                   <(tar xzfO nginx-chart/*.tgz nginx/values.yaml) \
                   <(tar xzfO rewritten-chart/chart.tgz nginx/values.yaml)
+
+
+  - name: test-pull-request
+    serial: true
+    plan:
+      - in_parallel:
+        - get: test-image
+          passed:
+            - build-test-image
+        - get: source
+          resource: pull-requests
+          trigger: true
+          version: every
+          params:
+            integration_tool: rebase
+      - put: pull-requests
+        inputs: detect
+        params:
+          path: source
+          status: pending
+      - task: run-unit-tests
+        image: test-image
+        file: source/ci/tasks/test.yaml
+        params:
+          MAKE_TARGET: test-units
+        on_failure:
```

Comment:
In the resource [upstream documentation](https://github.com/telia-oss/github-pr-resource) the example states

```
    on_failure:
      put: pull-request
      params:
        path: pull-request
        status: failure
```

Why is it that we set path:source while in the example use path:failure ?

Reply:
I understand now. The path is not the API path or anything like that. Is the name of the resource in the GET task